### ISSUE TITLE
Dutch language suggestions

### ISSUE THREAD

**Issue by jeroenmies** [NECROBOTIO/NecroBot#4619]
Can I suggest the following code changes to the Futch (translation.nl.json) language file?

{
      "Key": "farmPokestopsOutsideRadius",
      "Value":
        "Buiten het ingestelde bereik beland! Terug naar beginpunt ({0} meter weg) in 5 seconden. Klopt je LastPos.ini bestand?"
},
...
{
      "Key": "eventFortUsed",
      "Value": "XP: {0}, Gems: {1}, Items: {3}"
},
...
{
      "Key": "eventFortFailed",
      "Value": "Items verzamelen mislukt, mogelijke softban. Opnieuw proberen: {0}/{1}"
},
{
      "Key": "eventFortTargeted",
      "Value": "{0} over {1} meter en wordt bereikt over {2} seconden"
},

**Comment by noodle-** [comment]
Why don't you create a pull request?

Go here
https://github.com/NECROBOTIO/NecroBot/blob/master/PoGo.NecroBot.CLI/Config/Translations/translation.nl.json

Click on the edit button near the top, change whatever you want, and follow the instructions on screen to merge & create the pull request.

You can do all that stuff from your browser, good luck.

**Comment by theloser** [comment]
Speaking of version 0.8.5. there are a few very obvious errors in the "nl" language file. I don't speak dutch, so somebody else has to fix them, but here is one example (I saw this when I checked logfiles for issue #4170):

```
    {
      "Key": "eventFortUsed",
      "Value": "XP: {0}, Gems: {1}, Items: {2}"
    },

```
This produces the following output:

> [18:06:49] XP: The Falconer, Gems: 50, Items: 0

Nobody gets 50 gems and no items :-)
So, the correct form has to be:
```
    {
      "Key": "eventFortUsed",
      "Value": "Name: {0} XP: {1}, Gems: {2}, Items: {3}, Lat: {4}, Long: {5}"
    },
```
I suggest someone who speaks dutch checks this translation file and corrects it. Thanks!
